programs/pluto/spdb_struct.c

```diff
@@ -313,7 +313,8 @@ oakley_alg_makedb(struct alg_info_ike *ai
 	}
 	transcnt++;
     }
-    gsp->parentSA = TRUE;
+    if (gsp != NULL) gsp->parentSA = TRUE;
+    DBG(DBG_CONTROL, DBG_log("oakley_alg_makedb() returning %p", gsp));
```

Comment:
Returning a pointer value in a debug message means that it can never pass regression tests, because the pointer value can change.

Reply:
It will pass with commit #https://github.com/xelerance/Openswan/commit/89f85a27a5cf6867ce6a6b7e46cd86b2e5cad058